.devcontainer/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.217.4/containers/java/.devcontainer/base.Dockerfile
```

Comment:
If we're going to set up a dev environment here, do you think we should include a kafka instance with the appropriate topic to listen to on launch? And what about maybe kafkacat so we can push messages to that topic easily for testing?

Reply:
A kafka service has now been added.

Reply:
Kafkacat now gets installed through the dev container's dockerfile rather than being added as a service in the docker-compose file.